### ISSUE TITLE
Attributes on classes not displayed when using both properties and normal attributes 

### ISSUE THREAD

**Issue by charris** [numpy/numpydoc#12]
Copied from https://github.com/numpy/numpy/issues/421.

When documenting at the same time properties and normal attributes, nothing is rendered at all.

e.g. in the example below:

```
class ExampleClass1
    """
    Attributes
    ----------
    prop1
    attr2 : string 
        Some string attribute. 
    """
    @property 
    def prop1(self)
           """ Property 1 docsting """
           pass
```

The docstring is transformed in the following reST (docscrape_sphinx.py:57,strmember_list) :

```
.. rubric:: Attributes

.. autosummary::
   :toctree:

   prop1
==============  ================  ==========
        attr2             string  Some string attribute
==============  ================  ==========
```

but this is not correct as a newline is missing after "prop1":

**Comment by jnothman** [comment]
I think this must have been previously fixed and has been changed again in #106. It is tested in https://github.com/numpy/numpydoc/blob/f09abc8/numpydoc/tests/test_docscrape.py#L898 rendering as https://github.com/numpy/numpydoc/blob/f09abc8/numpydoc/tests/test_docscrape.py#L1028.

This should be closed.

**Comment by jnothman** [comment]
I'm closing
